### ISSUE TITLE
Support for multiple charts_dir's?

### ISSUE THREAD

**Issue by morremeyer** [helm/chart-releaser-action#90]
Hey everyone,

I’m writing this to get some feedback on an idea I currently have. 

TL;DR: How do you, especially the maintainers of the action, think about supporting multiple directories instead of just one `charts_dir`?

Longer explanation:

We’re currently managing multiple different parts of a software in seperate repositories. Charts are maintained in the same repository as the software they are written for to simplify lockstep releases. 

The aim currently is to have one central `charts` repository to enable users to

* only need to use one chart repository
* not clutter the releases of the software with the chart releases

However, to prevent any possible conflicts between charts, we currently sync the charts into the following structure in the `charts` repository:

```
charts/
  source_repository_name/
    chart_1/
      Chart.yaml
      templates
    chart_2/
      Chart.yaml
      templates
  another_source_repository/
    chart_3/
      Chart.yaml
      templates
```

Uniqueness of the chart names can be assumed here, but syncing to one `charts` directory directly can’t be achieved due to some duplicate file names (of needed files for packaging) on the `source_repository_name` level.

I tried running the chart-releaser-action on `charts/source_repository_name` first and then on `charts/another_source_repository`, this leads to:

* charts are packaged and released correctly
* **the `index.yaml` only contains the charts from the source repository for which the last release happened** <- this is not what we want.

I think we can avoid that by allowing the specification of multiple `charts_dir` directories.

What do you think? Any feedback is welcome.
